nwcsafpps_runner/utils.py

```diff
@@ -496,22 +496,21 @@ def get_pps_inputfile(platform_name, ppsfiles):
     return None
 
 
-def get_outputfiles(path, platform_name, orb, **kwargs):
-    """From the directory path and satellite id and orbit number scan the directory
-    and find all pps output files matching that scene and return the full
-    filenames. Since the orbit number is unstable there might be more than one
-    scene with the same orbit number and platform name. In order to avoid
-    picking up an older scene we check the file modifcation time, and if the
-    file is too old we discard it!
-
+def get_outputfiles(path, platform_name, orb, st_time, **kwargs):
+    """From the directory path and satellite id and orbit number and start time,
+    scan the directory and find all pps output files matching that scene and
+    return the full filenames. Since the orbit number is unstable there might be
+    more than one scene with the same orbit number and platform name. In order
+    to avoid picking up an older scene we check the file modifcation time, and
+    if the file is too old we discard it!
```

Comment:
Did you install the flake8-docstring plugin ? the docstyle in python says there should be one summary line and then the description after a blank line.

Reply:
Yes, but I did not go into details in functions that I did not create or spent a lot of time in.

Reply:
Uppdated in new PR